src/com/company/Main.java

```diff
@@ -2,37 +2,25 @@
 
 import java.time.LocalDateTime;
 import java.time.format.DateTimeFormatter;
-import java.util.Date;
-import java.util.Random;
-import java.util.Scanner;
+import java.util.*;
 
 import static java.time.temporal.TemporalQueries.localDate;
 
 public class Main {
 
     public static void main(String[] args) {
-
+        System.out.println("Type count of chat members");
         Scanner input = new Scanner(System.in);
-        int count = input.nextInt();
-        int activeMembersCount=count;
+        int membersCount = input.nextInt();
+        ArrayList<String> listOfChatMembers = new ArrayList<>(membersCount);
```

Comment:
pls try to use OOP here, I will suggest to Create User class which will have som properties(think what it can have ...) and use it, instead of ArrayList of String, in case of issues I will help:)

Reply:
Here I have some questions. I have only one field, user name, is it necessary to create a class, I have in my main class only one string list to resolve the problem of names. Thanks for your answer :)

Reply:
Lusine jan it is ok to use just string when you don't have many fields and/or methods needs to be stored somewhere, but you have passed OOP course so will be good be do tasks using separation of logic in some classes, so we can understand how you think on OOP level and if we ask to extend this task to more complex you, in any case, should go with having separate classes for users, but I agree this is also acceptable way for this task for now

Reply:
Diana jan, I will try to develop this problem for me, I will add new fields and methods and use OOP, but if you are not against, can I leave the task in this way for now? I also cleaned my master branch, I have all my changes in my feature branch. Thanks to review if everything is alright. I really thank you for your careful review :)

Reply:
@Lusin-mus  jan you can leave it as it is, as your solution is also acceptable, my comments are for making you (and others) to think how it can be done in other way based on experience and knowledge

Reply:
Diana jan, I really thank you a lot for your attention and your precious comments, for us as beginners it's important, because we learn a lot of things within the process :)